src/synopsys-action/synopsys-bridge.ts

```diff
@@ -91,6 +93,23 @@ export class SynopsysBridge {
         cwd: workingDirectory
       }
       try {
+        if (inputs.ENABLE_NETWORK_AIR_GAP) {
+          info('bridge default path :'.concat(this.getBridgeDefaultPath()))
+          info('synopsys_bridge_path:'.concat(inputs.SYNOPSYS_BRIDGE_PATH))
+          //const bridgepath = inputs.SYNOPSYS_BRIDGE_PATH.length == 0 ? this.getBridgeDefaultPath() : inputs.SYNOPSYS_BRIDGE_PATH
+          info('inputs.SYNOPSYS_BRIDGE_PATH.length::'.concat(inputs.SYNOPSYS_BRIDGE_PATH))
+          if (inputs.SYNOPSYS_BRIDGE_PATH.length !== 0) {
+            this.bridgeExecutablePath = await this.setBridgeExecutablePath(osName, inputs.SYNOPSYS_BRIDGE_PATH)
+            if (!checkIfPathExists(this.bridgeExecutablePath)) {
+              throw new Error('synopsys_bridge_path '.concat(this.synopsysBridgePath, ' does not exists'))
+            }
+          } else {
+            this.bridgeExecutablePath = await this.setBridgeExecutablePath(osName, this.getBridgeDefaultPath())
+            if (!checkIfPathExists(this.bridgeExecutablePath)) {
+              throw new Error('bridge_default_Path '.concat(this.synopsysBridgePath, ' does not exists'))
```

Comment:
we don't have any input variable like bridge_default_Path, can you make it **Synopsys Bridge default path does not exist**